### PR TITLE
[FSPE-5230] Add search bar component

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSPE-5230

### CODE REVIEW

File: src/website/views/components/forms/search-input/_examples.pug [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-disabled
+      li.-active
+        a(href=`#webcomponent-disabled`) Web Component
+      li
+        a(href=`#html-disabled`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-disabled
+    :code(lang="html")
+      <div class="chi-form__item">
+        <chi-search-input id="example__disabled" value="Sample Text" disabled="disabled"></chi-search-input>
```
Comment (janibolkvadze) [inline src/website/views/components/forms/search-input/_examples.pug:63]:
```suggestion
        <chi-search-input id="example__disabled" value="Sample Text" disabled></chi-search-input>
```

File: src/website/views/components/forms/search-input/_examples.pug [hunk 2]
```diff
@@ -0,0 +1,193 @@
+h2 Examples
+p.-text
+  | To render a search input, apply the class <code>chi-search__input</code> to an <code>input type="search"</code>.
```
Comment (janibolkvadze) [inline src/website/views/components/forms/search-input/_examples.pug:3]: Isn't this HTML blueprint specific message? I think for it's irrelevant for web-component tab

File: src/website/views/components/forms/search-input/_examples.pug [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            </div>
+          </button>
+        </div>
+      </div>
+
+h2 Sizes
+p.-text
+  | Search inputs supports the following sizes: <code>-sm</code>, <code>-md</code>,
+  |  <code>-lg</code>, <code>-xl</code>.
+  | The default size is <code>-md</code>.
```
Comment (janibolkvadze) [inline src/website/views/components/forms/search-input/_examples.pug:87]:
```suggestion
  | Search inputs supports the following sizes: <code>sm</code>, <code>md</code>,
  |  <code>lg</code>, <code>xl</code>.
  | The default size is <code>md</code>.
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by janibolkvadze** [comment]
I just tested including Chi assets from the remote instance of this PR with integrity attribute from the installation page and `chi.css` and `ux-chi-ce.esm.js` have been blocked by the browser. @jllr Please rebase from master, run `chi test` on your local. It will generate `sri.json` with updated SHA values (for `chi.css` and `ux-chi-ce.esm.js` because your changes affect these 2 files) and commit the updated `sri.son`.

File: src/custom-elements/src/components/search-input/search-input.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      onInput={(ev) => this._handleValueInput(ev)}
+      onChange={(ev) => this._handleValueChange(ev)}
+      autocomplete="off"
+      aria-label="search input"
+    />;
+
+    const searchXIcon = this._cleanButtonVisible ?
+      <button class={`
+        chi-button -icon -close
+        ${this.size === 'sm' ? '-sm' : ''}
```
Comment (mattnickles) [inline src/custom-elements/src/components/search-input/search-input.tsx:132]: I would suggest using the small X for all sizes. The large X served a different purpose which the UX team later omitted from the spec.

**Pull request closed without merging**